meilisearch-lib/src/index_controller/versioning/error.rs

```diff
@@ -1,6 +1,6 @@
 #[derive(thiserror::Error, Debug)]
 pub enum VersionFileError {
-    #[error("Version file is missing or the previous MeiliSearch engine version was below 0.24.0. Use a dump to update MeiliSearch.")]
+    #[error("Version file is missing or the previous MeiliSearch engine version was below 0.25.0. Use a dump to update MeiliSearch.")]
```

Comment:
> does this mean we have to change the version number in this message for each release?
No since we say `below v0.25.0`.

Reply:
I don't think so @curquiza, it's super abstract for them in my opinion. 

Couldn't we rather just tell the user to do a dump and be straight to the point? If I understand in this case the only solution is to make a dump.

Reply:
We could just say that we couldn't infer the version of the database instead?

Reply:
What do you think of these two proposals? 

`MeiliSearch (v0.25.0) couldn't start with the database files coming from a prior version. Please consider using a dump to load your data.`

`MeilSearch (v0.25.0) couldn't start from a prior version database files. Please consider using a dump to load your data.`

`MeilSearch (v0.25.0) failed to infer the version of the database. Please consider using a dump to load your data.`

Reply:
There is another message for when MeiliSearch can't start « from a prior version database » where we specify the database number etc.
So I vote for the last error message, which seems more relevant to me:
```
MeilSearch (v0.25.0) failed to infer the version of the database. Please consider using a dump to load your data.
```
Also, if someone comes asking for help because he corrupted its `VERSION` file, it'll be easier for us to explain the error with this error message.

Reply:
Agreed